cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/preview/DefaultPreviewManager.java

```diff
@@ -134,8 +134,7 @@
 
     // TODO make maximum size and expire after write configurable?
     this.appInjectors = CacheBuilder.newBuilder()
-      .maximumSize(10)
-      .expireAfterWrite(15, TimeUnit.MINUTES)
+      .maximumSize(cConf.getInt(Constants.Preview.PREVIEW_CACHE_SIZE, 10))
```

Comment:
Curious, shouldn't the default value be placed in cdap-default.xml instead of relying on cdap-site.xml?